charts/xui-webapp/values.preview.template.yaml

```diff
@@ -4,6 +4,30 @@ nodejs:
   environment:
        IDAM_URL: https://idam-api.aat.platform.hmcts.net
        S2S_URL: http://rpe-service-auth-provider-aat.service.core-compute-aat.internal
+       APPINSIGHTS_INSTRUMENTATIONKEY: ${APPINSIGHTS_INSTRUMENTATIONKEY}
```

Comment:
this should reside in the keyVault (and pulled through as an env variable)

Reply:
Please wait for the next PR.